src/Files.App/Views/SettingsPages/Appearance.xaml.cs

```diff
@@ -56,5 +56,15 @@ private async void ViewModel_PropertyChanged(object sender, System.ComponentMode
                 }
             }
         }
+
+        private void AppThemeSelectionGridView_Loaded(object sender, RoutedEventArgs e)
+        {
+            AppThemeLoading.Visibility = Visibility.Visible;
+            DispatcherQueue.TryEnqueue(async () =>
+            {
+                await App.ExternalResourcesHelper.LoadOtherThemesAsync();
+                AppThemeLoading.Visibility = Visibility.Collapsed;
```

Comment:
Would it be better to create a property in the view model for this?

Reply:
You're right.